.rpm_version

```diff
@@ -1 +1 @@
-1.12.14
+1.12.15
```

Comment:
Since this is another fix for a JIRA that we already have marked in the 1.12.14 release, I think we should just leave this as 1.12.14 and move the tag.

Reply:
fixed